### PR TITLE
koordlet: require gc started for cacheable update

### PR DESCRIPTION
**Pull request by saintube**
### Ⅰ. Describe what this PR does

To avoid memory leaks, require the cache GC to be started when the executor invokes cache updating.

### V. Checklist

- [X] I have written necessary docs and comments
- [x] I have added necessary unit tests and integration tests
- [X] All checks passed in `make test`

### CODE REVIEW

File: pkg/koordlet/resourceexecutor/executor.go [hunk 1]
```diff
@@ -110,6 +121,11 @@ func (e *ResourceUpdateExecutorImpl) UpdateBatch(cacheable bool, updaters ...Res
 func (e *ResourceUpdateExecutorImpl) LeveledUpdateBatch(cacheable bool, updaters [][]ResourceUpdater) {
```
Comment (zwzhang0107) [inline pkg/koordlet/resourceexecutor/executor.go:121]: remove cacheable since it is never supported?

Comment (saintube) [reply pkg/koordlet/resourceexecutor/executor.go]: done

**Pull request merged**